dim/models/tests/base.py

```diff
@@ -42,12 +41,11 @@ def generate_test_sql(self, test_type):
 
         target_file = generated_sql_folder.joinpath(f'{test_type}.sql')
         generated_sql = template.render(self.config)
-
         sql_to_file(target_file=target_file, sql=generated_sql)
 
         return target_file, generated_sql
 
     def execute_test_sql(self, sql):
         # TODO: extract authentication logic into separate module
-        client = bigquery.Client.from_service_account_json(json_credentials_path=CREDS, project=GCP_PROJECT)
+        client = bigquery.Client(project=GCP_PROJECT)
```

Comment:
Does this also work with creds file? Once this is running in a container we will want to utilize the json here.